docs/content/concepts/sui-move-concepts/packages/upgrade.mdx

```diff
@@ -496,7 +536,23 @@ Array [
 
 ```
 
-The result provides a new ID for the upgraded package. As was the case before the upgrade, you need to include that information in your manifest so any of your other packages that depend on your `sui_package` know where to find the on-chain bytecode for verification. Edit your manifest once again to provide the upgraded package ID for the `published-at` value, and return the original `sui_package` ID value in the `[addresses]` section:
+The result provides a new ID for the upgraded package.
+
+<Tabs>
+
+<TabItem value="automated-address-management" label="Automated Addresses">
+
+After upgrading, return the original `sui_package` ID value in the `[addresses]` section:
+
+```toml
+[addresses]
+sui_package = "<ORIGINAL-PACKAGE-ID>"
+```
+</TabItem>
+
+<TabItem value="manual-address-management" label="Manual Addresses">
+
+So that any of your other packages that depend on your `sui_package` know where to find the on-chain bytecode for verification, edit your manifest once again to provide the upgraded package ID for the `published-at` value, and return the original `sui_package` ID value in the `[addresses]` section:
```

Comment:
```suggestion
So that packages that depend on your `sui_package` know where to find the on-chain bytecode for verification, edit your manifest again. Provide the upgraded package ID for the `published-at` value and return the original `sui_package` ID value in the `[addresses]` section:
```
Just suggesting to break up the longer sentence.